web/src/components/ArrowTooltip.tsx

```diff
@@ -19,13 +21,47 @@ const useStylesBootstrap = makeStyles((theme: Theme) => {
 export const ArrowTooltip = (props: TooltipProps): ReactElement => {
   const classes = useStylesBootstrap();
 
+  const isMobile = useMediaQuery(MediaQueries.isMobile);
+
+  const [open, setOpen] = useState(false);
+
   return (
-    <Tooltip
-      arrow
-      enterTouchDelay={0}
-      classes={classes}
-      {...props}
-      onOpen={analytics.event.tooltip.mouseover.view_tooltip}
-    />
+    <>
+      {isMobile ? (
+        <ClickAwayListener onClickAway={(): void => setOpen(false)}>
+          <div className="display-flex">
+            <Tooltip
+              arrow
+              enterTouchDelay={0}
+              classes={classes}
+              onOpen={analytics.event.tooltip.mouseover.view_tooltip}
+              PopperProps={{
+                disablePortal: true,
+              }}
+              onClose={(): void => setOpen(false)}
+              open={open}
+              disableFocusListener
+              disableHoverListener
+              disableTouchListener
+              {...props}
+            >
+              <div>
```

Comment:
div wrapper is needed to make the MUI Tooltip component happy with its children.